apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/converter/TimeDurationValueConverter.java

```diff
@@ -63,4 +68,8 @@ public String toString(TimeDuration value) {
     public String getDefaultDurationSuffix() {
         return defaultDurationSuffix;
     }
+
```

Comment:
When removing default durations from the docs, we can probably remove this method, too if it then doesn't have any callers anymore.

```suggestion
```